dts/bindings/arm/nordic,nrf-uicr-v2.yaml

```diff
@@ -0,0 +1,26 @@
+# Copyright (c) 2024 Nordic Semiconductor ASA
+# SPDX-License-Identifier: Apache-2.0
+
+description: Nordic UICR v2 (User Information Configuration Registers)
```

Comment:
I'd suggest splitting bindings into its own commits, since this one is large enough.

Reply:
Done.